new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -67,6 +69,6 @@ private String buildDownloadCommand() {
     
     return "curl -fL " 
```

Comment:
i think this whole method could be better written as 
```
String command = String.format("...", a, b, c, d);
return command;
```

Reply:
String.format does make it easier to read and less error-prone.

Reply:
done